src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx

```diff
@@ -160,7 +164,7 @@ export class NotificationConsoleComponent extends React.Component<
                 role="combobox"
                 selectedKey={this.state.selectedFilter}
                 options={NotificationConsoleComponent.FilterOptions}
-                onChange={this.onFilterSelected.bind(this)}
+                onChange={() => this.onFilterSelected}
```

Comment:
This is a behavior change. You are not calling `this.onFilterSelected`

`this.onFilterSelected()`

Reply:
My fault, I see the onFilterSelected is already arrow function, anyway, I just want to away bind and use arrow function